WSL/install.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Install WSL
+description: Install Windows Subsystem for Linux with the command, wsl --install. Use a Bash terminal on your Windows machine run by your preferred Linux distribution - Ubuntu, Debian, SUSE, Kali, Fedora, Pengwin, Alpine, and more are available.
+keywords: wsl, install, BashOnWindows, bash, windows subsystem for linux, install ubuntu on windows, enable WSL2, linux on windows
+ms.date: 09/10/2021
+ms.topic: article
+ms.localizationpriority: high
+---
+
+# Install WSL
+
+You can now install everything you need to run Windows Subsystem for Linux (WSL) by entering this command in PowerShell or Windows Command Prompt and then restarting your machine.
+
+```powershell
+wsl --install
+```
+
+> [!IMPORTANT]
```

Comment:
Can we move this section up and have it as a heading section of: **Pre-requisites** instead of a note?

The reasoning here is that we already get tons of issues of people saying the command doesn't work, I'd want to make sure that the pre-reqs are at the top so people read them before they start trying to run commands